tests/packages/mlx/test_mellanox-modules-compat.py

```diff
@@ -0,0 +1,27 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): any master host of a pool, with access to XCP-ng RPM repositories.
+import logging
+
+from pkgfixtures import host_with_saved_yum_state
+
+MLX4_MODULE = 'mlx4_en'
+
+def load_unload_mlx_module(host):
+    logging.info("loading mlx module")
+    host.ssh(['modprobe -v', MLX4_MODULE])
+    logging.info("unloading mlx module")
+    host.ssh(['modprobe -r', MLX4_MODULE])
```

Comment:
Let's be also verbose when we unload the module?
```suggestion
    host.ssh(['modprobe -rv', MLX4_MODULE])
```